components/Header.vue

```diff
@@ -1,14 +1,14 @@
 <template>
-  <header class="w-full p-4 shadow-md sticky top-0 bg-surface-primary z-10">
-    <div class="prose-a:no-underline max-w-7xl mx-auto flex items-center justify-start space-x-2">
+  <header class="not-prose w-full p-4 shadow-md sticky top-0 bg-surface-primary z-10">
+    <div class="max-w-7xl mx-auto flex items-center justify-start space-x-2">
       <NuxtLink
         to="/"
         class="flex items-center"
       >
         <icons-centrapay-logo class="icon-3xl fill-brand-accent" />
-        <h1 class="hidden md:block type-headline-4 px-1">
+        <span class="hidden md:block type-headline-4 px-1">
```

Comment:
I had another look at this after I said it should be a `h1` - I think that feedback was incorrect sorry @Wulingoh <3
There should only be one h1 per page which will be in the content.